.github/workflows/cleanup-staging-packages.yml

```diff
@@ -0,0 +1,30 @@
+name: 👋 staging ⬢🗑️
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  ghcr:
+    runs-on: ubuntu-22.04
+    if: github.event.pull_request.merged == true
+    timeout-minutes: 15
+    steps:
+      - name: Inspect the PR published package
+        id: inspect-package
+        run: |
+          gh api --paginate \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /users/kachick/packages/container/home/versions \
+            --jq '.[] | select(.metadata.container.tags[] | contains("pr-${{ github.event.pull_request.number }}")).id' | \
```

Comment:
Based the pr-NUM tag is not enough, that tag will be updated by the lates commit in the PR 🤔